R/fit_model.R

```diff
@@ -160,15 +160,24 @@ format_right_truncation <- function(pmf, data) {
     # silently removed if length of the PMF was longer than the data,
     # effectively eliminating the right-truncation correction
 
-    cli::cli_abort(
+    trunc_idx <- nrow(data) - 1
```

Comment:
Is this an `idx`? Isn't `seq_len(trunc_idx)` the `idx` and this is a length?

Reply:
I don't think so? The index refers to the location of the specific element in the list or vector. `seq_len(trunc_idx)` constructs a vector of indices from the index.

Reply:
I think this is a software jargon != math jargon thing? See e.g., [python docs](https://www.w3schools.com/python/ref_list_index.asp) on this

Reply:
I don't think it's a software/math difference thing. This ultimately doesn't matter much. The index is the position that something occurs. You want to truncate the PMF to be shorter than the data. You could say that what you want is to specify the highest index of the PMF. You could also call that the length of the PMF. I find calling it the length less confusing. Feel free to action or to close as you see fit

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/55/commits/69ba3931e42eb999667494de747d60bca6a3aa8c